client/src/components/Counter.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ReactComponent as HeartIcon } from "../assets/icons/heart-icon.svg";
+import styled from "@emotion/styled";
+
+export default function Counter(props) {
+  return (
+    <CounterStyle>
+      <Heart>
+        <HeartIcon />
+      </Heart>
+      <output>{props.number}</output>
```

Comment:
If you would destructre, you could write your code like this:
```suggestion
      <output>{number}</output>
```

Reply:
Cheers, just implemented!